src/api.js

```diff
@@ -84,3 +84,17 @@ export const loginUser = async user => {
     throw error;
   }
 };
+export const logoutUser = async token => {
+  try {
+    const response = await axios.delete(
+      "http://localhost:8080/auth/logout",
+      token
+    );
+    toast.success("Logout successful!");
+    return response.status;
```

Comment:
Out of curiosity do you expect to do anything this the status code? As I can't see you using it anywhere

Reply:
I don't think we use it anywhere at the moment and I can't think of a reason to use it later on, should we remove this for now as it would be very easy to add back in if we do end up needing it?

Reply:
If you're not using it, there's no need to really return that value :)